docs/installation/local.md

```diff
@@ -92,6 +92,17 @@ Now to activate the virtual environment, type
 pipenv shell
 ```
 
+After installing dependencies in your virtual environment, you need to configure pre-commit hooks by running the command
+
+```sh
+pre-commit install
+```
+If you get command not found errors you can install pre-commit in your virtual environment. Type,
+
+```sh
+pip install pre-commit
+```
```

Comment:
This shouldn't be necessary as pre-commit in present in requirements

Reply:
That's correct. Will remove, I had added it in case of errors.